src/commcare_cloud/cli_utils.py

```diff
@@ -8,12 +8,16 @@
 from six.moves import input, shlex_quote
 
 
-def ask(message, strict=False, quiet=False):
+def ask(message, strict=False, quiet=False, default=False):
     if quiet:
         return True
     yesno = 'YES/NO' if strict else 'y/N'
-    negatives = ('NO', 'N', 'n', 'no', '')
+    negatives = ('NO', 'N', 'n', 'no')
     affirmatives = ('YES',) if strict else ('y', 'Y', 'yes')
+    if default:
```

Comment:
I think this whole `if`/`else` construct should be wrapped with `if not strict:`

Reply:
Thanks that's a good point. Once I got into the spirit of this suggestion, I realized the interaction between the strict, quiet, and default args wasn't that well defined, so I gave each of them a job to carry out dutifully instead of each one making assumptions about the other arg (e.g. `quiet` and `strict` seemed to at least partially presuppose default=True).  It's true I think the `default` is unused at this point, but I was about to use it before I decided to stop where I was and PR it. (The use case I have in mind is to prompt on whether to deploy formplayer in the `deploy` command, but to default to "no".) Thanks for the feedback, let me know whether you're happy with my solution/approach.

Reply:
I ended up changing my mind. Here's my reasoning pasted from the last commit ("Undo recent changes to ask")
> I think I was thinking about this the wrong way, and I decided I don't
actually want to touch this. I realized we rely on ask having a
particular meaning: it means "here's something that you should say yes to
if you want to do the normal thing". To present a consistent experience,
if we want the default to be no, we should just ask the question in the
negative: "Do you want to skip formplayer deploy? [Y/n]". This allows
us to always pick a predictable answer when using --quiet, which means "say
yes to everything". In the previous commit where I changed quiet to
always pick the default, that was obviously a problem because our
default is current No everywhere, but we want quiet to pick Yes for you.

Reply:
I ended up deciding against all this. I wrote up my rationale in the last commit (I was sloppy with my example and so the logic there doesn't quite make sense, but I think the overall sentiment was right).